packages/web/docs/src/pages/product-updates/2024-06-14-ms-teams-webhooks-alerts.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: MS Teams Webhooks Alerts
+description: Configure MS Teams webhooks to receive alerts and notifications from Hive.
+date: 2024-06-14
+authors: [jiri]
+---
+
+We added support for configuring MS Teams webhooks to receive alerts and notifications from Hive.
+
+![MS Teams Webhook dialog](/changelog/2024-06-14-ms-teams-webhooks-alerts/project-msteams-alert.png)
```

Comment:
Let's use a picture that shows the remaining app in the background, so it is not just one big modal covering the full-screen width. Similar to https://the-guild.dev/graphql/hive/product-updates/2024-04-09-approval-comment